### ISSUE TITLE
TPS issue in Multiplayer - Rebirth of the Night modpack

### ISSUE THREAD

**Issue by kil0byt3** [Ordinastie/MalisisDoors#353]
Hello, I'm a dev for the [Rebirth of the Night](https://www.curseforge.com/minecraft/modpacks/rebirth-of-the-night) modpack. We're seeing some major TPS issues in our modpack that I think may be related to MalisisDoors/Core.

Using LagGoggles:
Before removing Malisis Doors/Core:
![image](https://user-images.githubusercontent.com/39413400/78465962-f120c600-76b0-11ea-8fa0-fa5c58e16e2a.png)

After removing Malisis Doors/Core:
![image](https://user-images.githubusercontent.com/39413400/78465963-f54ce380-76b0-11ea-94ba-f82de090120c.png)

Is there any reason this TPS loss/server thread usage would be so high for MalisisCore?
I think MalisisCore also handles some raytracing or collision things that might conflict with something or cascade something?

I can provide logs/more info if needed, let me know.

Thanks!

**Comment by CalaMariGold** [comment]
Yeah MalisisCore also causes a massive amount of cascading world gen too, which is strange because I don't know why it would need to change anything with world-gen... We're just using Malisis Doors.

**Comment by InuYasha86000** [comment]
this might be the same issue i'm having with my own custom modpack, my server keeps shutting down and saying that a tick took 60 seconds and so the server classes that as a crash, here's the crash report
[crash-2020-04-07_17.47.05-server.txt](https://github.com/Ordinastie/MalisisDoors/files/4448529/crash-2020-04-07_17.47.05-server.txt)
